bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "browser-store",
   "dependencies": {
-    "amplify": "~1.1.2",
-    "js-cookie": "~2.1.1"
+    "amplify": "~1.1.2"
```

Comment:
Heh... I've been thinking about trimming down this entire lib. This PR will make things a lot easier.